README.md

```diff
@@ -4,17 +4,13 @@
   | ------------- | ------------- |
   | <img width="426" alt="image" src="https://github.com/xyzeva/blocky-web/assets/133499694/31fa8618-2010-4200-91ef-5211129cb51d"> | <img width="426" alt="image" src="https://github.com/xyzeva/blocky-web/assets/133499694/20a78fa1-5403-4ae7-84dd-0c23a37b53d5"> |
 
-  <h1>blocky web</h1>
-  <p>a userscript to bring back the boxiness of the old web for *any* website</p>
```

Comment:
don't change the readme. ideally just add the extension code and i will handle publishing